collectors/APICalls/breakpoints.js

```diff
@@ -96,6 +123,16 @@ const breakpoints = [
             {name: 'interestCohort'} // FLoC
         ]
     },
+    {
+        proto: 'CookieStore', // modern version of document.cookie
+        props: [
+        ],
+        methods: [
+            {name: 'get'},
+            {name: 'getAll'},
+            {name: 'set', saveArguments: true},
```

Comment:
@jdorweiler so cookies can be set using this method IN ADDITION TO `document.cookie = 'bla'`. Do we need to update something in detector?

Reply:
We have this https://github.com/duckduckgo/tracker-radar-detector/blob/main/src/trackers/classes/request.js#L60

Reply:
I see, we need to look for `CookieStore set` too.